### ISSUE TITLE
Please use Windows /temp folders for temporary files - move them out of /Documents

### ISSUE THREAD

**Issue by dsnvwlmnt** [PoE-TradeMacro/POE-TradeMacro#1136]

**Comment by Techie-Fox** [comment]
YES! Argh I cannot tell you how much it BUGS me that this presumptuously dumps all its config into my Documents folder. At the very least it could use Documents\My Games, but should probably use AppData rather than Temp if going that route.

At this rate I think I'm just gonna uninstall, that might seem harsh over a trivial issue but I'm not exactly serious with the game, and it really is annoying.

**Comment by dsnvwlmnt** [comment]
I'm ok with it using Documents or My Games for "user settings", but temp files the client generates should not be in Documents.

**Comment by Eruyome** [comment]
What temporary files are you talking about? The ones inside the macros `mainfolder\temp`?

**Comment by dsnvwlmnt** [comment]
Any temporary files. I'm guessing that's what I was referring to given that it's called temp, but I don't recall. If there are other temp folders those can go too.
